plone/app/z3cform/templates/image_input.pt

```diff
@@ -46,108 +28,79 @@
     </span>
   </tal:if>
   <span tal:condition="python:exists and download_url and action=='nochange'">
-    <a tal:attributes="
```

Comment:
Also here, some lines above there is a `${view/value/filename}`. Is it different from the already defined `filename`?

Reply:
done